boa/src/syntax/ast/node/labelled_smt.rs

```diff
@@ -0,0 +1,38 @@
+use super::Node;
+use gc::{Finalize, Trace};
+use std::fmt;
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Clone, Debug, Trace, Finalize, PartialEq)]
+pub struct Label {
```

Comment:
I would call this with the same name as in the spec: `LabelledStatement`, especially because it could be seen as if this were just the label, without the statement.